app/src/renderer/components/common/ActionModal.vue

```diff
@@ -181,20 +224,44 @@ export default {
     },
     submissionErrorPrefix: {
       type: String,
-      default: `Submitting data failed`
+      default: `Transaction failed`
+    },
+    amount: {
+      type: [String, Number],
+      default: `0`
     }
   },
   data: () => ({
     step: defaultStep,
     signMethod: null,
     password: null,
     sending: false,
+    gasEstimate: null,
+    gasPrice: (0.025 * 1e-6).toFixed(9), // recomended default: 0.025 uatom per gas
```

Comment:
can we extract this constant into `config.js`?

Reply:
updated